en/db-odm.md

```diff
@@ -425,7 +428,7 @@ echo 'The generated id is: ', $robot->getId();
 ### Validation Messages
 `Phalcon\Mvc\Collection` has a messaging subsystem that provides a flexible way to output or store the validation messages generated during the insert/update processes.
 
-Each message consists of an instance of the class `Phalcon\Mvc\Model\Message`. The set of messages generated can be retrieved with the method getMessages(). Each message provides extended information like the field name that generated the message or the message type:
+Each message consists of an instance of the class `Phalcon\Mvc\Model\Message`. The set of messages generated can be retrieved with the method `getMessages()`. Each message provides extended information like the field name that generated the message or the message type:
```

Comment:
Heck, I would love to update the parser so patterns that look like this automatically produce backlinks to the api documentation. `Phalcon\Mvc\Model\Message`